packages/terra-date-picker/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Unreleased
 ### Changed
 * Changed `onChange` and `onChangeRaw` callbacks to return the following metadata as part of the third parameter: iso, inputValue, isCompleteValue, isValidValue
 * Update the placeholder for the `es-US`, `nl`, `sv-SE`, and `sv` locales to match the actual date format.
+* Applied new border styles for keyboard navigation.
+* Changed pre-selected date to 1st of month when changing year/month.
```

Comment:
Move these changelog entries to the unreleased section

Reply:
Moved 64aef1f